<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="verticalFrame">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>0</y>
      <width>241</width>
      <height>491</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
* { background: rgb(203, 255, 209); }

</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>CH1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
       </property>
       <property name="margin">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_1">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Volt CH1</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="voltage_ch1">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="suffix">
          <string> V</string>
         </property>
         <property name="maximum">
          <double>32.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Presets Output Voltage</string>
         </property>
         <widget class="QRadioButton" name="preset_ch1_voltage_3v3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>81</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>3.3 V</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="preset_ch1_voltage_12v0">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>81</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>12 V</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="preset_ch1_voltage_13v8">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>40</y>
            <width>119</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>13.8 V</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="preset_ch1_voltage_5v0">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>20</y>
            <width>119</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>5.0 V</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Amp:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="margin">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="current_ch1">
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::PlusMinus</enum>
         </property>
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="suffix">
          <string> A</string>
         </property>
         <property name="maximum">
          <double>3.200000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Presets Current Limit</string>
         </property>
         <widget class="QRadioButton" name="preset_ch1_current_0a1">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>20</y>
            <width>119</width>
            <height>28</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>Immediately set CH1 current limit to 100 mA</string>
          </property>
          <property name="text">
           <string>100 mA</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="preset_ch1_current_0a5">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>40</y>
            <width>119</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>500 mA</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="preset_ch1_current_1a0">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>20</y>
            <width>119</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>1 A</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="preset_ch1_current_2a0">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>40</y>
            <width>119</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>2 A</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>Output On</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="verticalFrame_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>241</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">* { background: rgb(255, 246, 192); }
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,0,0,0">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>CH2</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
       </property>
       <property name="margin">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Volt:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="margin">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="voltage_ch2">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::PlusMinus</enum>
           </property>
           <property name="accelerated">
            <bool>true</bool>
           </property>
           <property name="suffix">
            <string> V</string>
           </property>
           <property name="maximum">
            <double>32.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Presets Output Voltage</string>
         </property>
         <widget class="QRadioButton" name="preset_ch2_voltage_3v3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>81</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>3.3 V</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="preset_ch2_voltage_12v0">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>81</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>12 V</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="preset_ch2_voltage_13v8">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>40</y>
            <width>119</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>13.8 V</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="preset_ch2_voltage_5v0">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>20</y>
            <width>119</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>5.0 V</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Amp:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="margin">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="current_ch2">
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::PlusMinus</enum>
         </property>
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="suffix">
          <string> A</string>
         </property>
         <property name="maximum">
          <double>3.200000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Presets Current Limit</string>
         </property>
         <widget class="QRadioButton" name="preset_ch2_current_0a1">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>20</y>
            <width>119</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>100 mA</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="preset_ch2_current_0a5">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>40</y>
            <width>119</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>500 mA</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="preset_ch2_current_1a0">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>20</y>
            <width>119</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>1 A</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="preset_ch2_current_2a0">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>40</y>
            <width>119</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>2 A</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_2">
         <property name="text">
          <string>Output On</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>-10</y>
      <width>31</width>
      <height>471</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>531</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSetup">
    <property name="title">
     <string>Setup</string>
    </property>
    <addaction name="actionIP_Address"/>
    <addaction name="actionUSB_Serial_Port"/>
    <addaction name="separator"/>
    <addaction name="actionStatus"/>
   </widget>
   <addaction name="menuSetup"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionIP_Address">
   <property name="text">
    <string>IPv4 networking</string>
   </property>
  </action>
  <action name="actionStatus">
   <property name="text">
    <string>Status</string>
   </property>
  </action>
  <action name="actionUSB_Serial_Port">
   <property name="text">
    <string>USB Serial Port</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
